src/EnergyPlus/ChilledCeilingPanelSimple.cc

```diff
@@ -0,0 +1,1571 @@
+				ShowFatalError( "Preceding condition causes termination." );
+
+			}}
+
+			PowerMet = CoolingPanel( CoolingPanelNum ).TotPower;
+
+			UpdateCoolingPanel( CoolingPanelNum );
+
+			ReportCoolingPanel( CoolingPanelNum );
+
+		} else {
+			ShowFatalError( "SimCoolingPanelSimple: Unit not found=" + EquipName );
+		}
+
+	}
+
+	void
+	GetCoolingPanelInput()
+	{
+
+		// SUBROUTINE INFORMATION:
+		//       AUTHOR         Rick Strand
+		//       DATE WRITTEN   Aug 2014
+		//       MODIFIED       na
+		//       RE-ENGINEERED  na
+
+		// PURPOSE OF THIS SUBROUTINE:
+		// This subroutine gets the input for the simple cooling panel units.
+
+		// METHODOLOGY EMPLOYED:
+		// Standard input processor calls--started from Daeho's radiant-convective water baseboard model.
+
+		// REFERENCES:
+		// na
+
+		// Using/Aliasing
+		using DataLoopNode::Node;
+		using DataLoopNode::NodeType_Water;
+		using DataLoopNode::NodeConnectionType_Inlet;
+		using DataLoopNode::NodeConnectionType_Outlet;
+		using DataLoopNode::ObjectIsNotParent;
+		//unused0909    USE DataGlobals,           ONLY: NumOfZones
+		using InputProcessor::GetNumObjectsFound;
+		using InputProcessor::GetObjectItem;
+		using InputProcessor::FindItemInList;
+		using InputProcessor::SameString;
+		using InputProcessor::VerifyName;
+		using NodeInputManager::GetOnlySingleNode;
+		using BranchNodeConnections::TestCompSet;
+		using DataSurfaces::Surface;
+		using DataSurfaces::TotSurfaces;
+		using ScheduleManager::GetScheduleIndex;
+		using ScheduleManager::GetCurrentScheduleValue;
+		using General::RoundSigDigits;
+		using General::TrimSigDigits;
+		using namespace DataIPShortCuts;
+
+		// Locals
+		// SUBROUTINE ARGUMENT DEFINITIONS:
+		// na
+
+		// SUBROUTINE PARAMETER DEFINITIONS:
+		static std::string const RoutineName( "GetCoolingPanelInput:" );
+		Real64 const MaxFraction( 1.0 );
+		Real64 const MinFraction( 0.0 );
+		Real64 const MaxWaterTempAvg( 30.0 ); // Maximum limit of average water temperature in degree C
+		Real64 const MinWaterTempAvg( 0.0 ); // Minimum limit of average water temperature in degree C
+		Real64 const HighWaterMassFlowRate( 10.0 ); // Maximum limit of water mass flow rate in kg/s
+		Real64 const LowWaterMassFlowRate( 0.00001 ); // Minimum limit of water mass flow rate in kg/s
+		Real64 const MaxWaterFlowRate( 10.0 ); // Maximum limit of water volume flow rate in m3/s
+		Real64 const MinWaterFlowRate( 0.00001 ); // Minimum limit of water volume flow rate in m3/s
+		Real64 const WaterMassFlowDefault( 0.063 ); // Default water mass flow rate in kg/s
+		//    INTEGER, PARAMETER   :: MaxDistribSurfaces    = 20         ! Maximum number of surfaces that a baseboard heater can radiate to
+		int const MinDistribSurfaces( 1 ); // Minimum number of surfaces that a baseboard heater can radiate to
+		Real64 const MinThrottlingRange( 0.5 ); // Smallest throttling range allowed in degrees Celsius
+		static std::string const MeanAirTemperature( "MeanAirTemperature" );
```

Comment:
@RKStrand These don't need to be static.  In fact, they don't seem useful at all.  They are only used once, so why not put the literal strings in the ifs below?